core/pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>puppetlabs</groupId>
+    <groupId>puppetlabs.org.jruby</groupId>
     <artifactId>jruby-parent</artifactId>
     <version>1.7.26-puppetlabs-1-SNAPSHOT</version>
```

Comment:
Yeah, maybe `-rc1` might be a good compromise.  That way we could reserve `-1`, etc., for any release we might want to do later that would come right off the corresponding upstream release tag.